pkg/server/commands/list_objects.go

```diff
@@ -48,6 +50,7 @@ type listObjectsRequest interface {
 func (q *ListObjectsQuery) evaluate(
 	ctx context.Context,
 	req listObjectsRequest,
+	typesys *typesystem.TypeSystem,
```

Comment:
Similar comment as above. `typesys` is a concrete instance of a model ID. It can be part of `listObjectsRequest`

Reply:
See https://github.com/openfga/openfga/pull/756#discussion_r1199491826